course/templatetags/coursetags.py

```diff
@@ -36,8 +36,8 @@ def __init__(self, variable):
         self.variable = variable
 
     def render(self, context):
-        js_lang_name = to_js_lang_name(translation.get_language())
-        context[self.variable] = js_lang_name
+        datatables_lang_name = to_datatables_lang_name(translation.get_language())
+        context[self.variable] = datatables_lang_name
```

Comment:
Since this is refactored, shall we also rename the following `get_current_js_lang_name` to explicitly indicate it's for datatable js only?